src/main/java/com/hms_networks/americas/sc/extensions/datapoint/DataQuality.java

```diff
@@ -62,8 +154,29 @@ private DataQuality(byte instanceDataQuality) {
    * Returns the constant byte representing the data quality.
    *
    * @return byte representing data quality
+   * @since 1.0.0
    */
   public byte getRawDataQuality() {
     return instanceDataQuality;
   }
+
+  /**
+   * Returns a string representation of the data quality.
+   *
+   * @return string representation of data quality
+   * @since 1.1.0
+   */
+  public String toString() {
+    String qualityString;
+    if (this == GOOD) {
+      qualityString = DATA_QUALITY_GOOD_STRING;
+    } else if (this == BAD) {
+      qualityString = DATA_QUALITY_BAD_STRING;
+    } else if (this == UNCERTAIN) {
+      qualityString = DATA_QUALITY_UNCERTAIN_STRING;
+    } else {
+      qualityString = DATA_QUALITY_UNKNOWN_STRING;
```

Comment:
Should this be uncertain? Are we introducing a new state "unknown" that is really unnecessary because there is already "uncertain".

Reply:
Uncertain is an actual, valid, quality value. Unknown is the absence of a known quality value.

Reply:
![image](https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/assets/51409174/7a3830d3-9ea9-4168-8bb8-4a23e8835d8f)